main.py

```diff
@@ -18,19 +21,27 @@ def read_env_variables():
     global TIMEOUT, EMAIL, PASSWORD, LOGLEVEL, SLEEPTIME, TOTP, DEBUG
 
     # dev environment
-    DEBUG = os.getenv('DEBUG') is not None
+    DEBUG = True
 
     value = os.getenv('TIMEOUT') or 20
     TIMEOUT = int(value)
 
-    EMAIL = os.getenv('EMAIL') or ''
-    PASSWORD = os.getenv('PASSWORD') or ''
+    EMAIL = "slinucs@gmail.com"
+    PASSWORD = "30Matvid11"
```

Comment:
Be careful with adding credentials to the commit by accident!

Reply:
Oh my, I didn't notice, thanks